applications/asset_tracker/src/main.c

```diff
@@ -1750,6 +1750,51 @@ void handle_bsdlib_init_ret(void)
 	#endif /* CONFIG_BSD_LIBRARY */
 }
 
+static void lte_handler(const struct lte_lc_evt *const evt)
+{
+	#if defined(CONFIG_BSD_LIBRARY)
```

Comment:
Perhaps have `ifdef` and the later `endif` without indentation for consistency and readability